namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-dev/resources/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "business_unit" {
 
 variable "team_name" {
   description = "The name of your development team"
-  default     = "crime-apps-team"
+  default     = "crime-apps"
```

Comment:
Changing this will destroy existing queues and add new ones.
https://concourse.cloud-platform.service.justice.gov.uk/builds/3669
Can you look at the changes in this plan pipeline and see if you are ok with the changes

Reply:
new queue names should be ok and changes should be fine.